diff --git a/features/src/main/javascript/features/rpc/rpc.js b/features/src/main/javascript/features/rpc/rpc.js
index 06b5cf7..9f70e69 100644
--- a/features/src/main/javascript/features/rpc/rpc.js
+++ b/features/src/main/javascript/features/rpc/rpc.js
@@ -369,10 +369,10 @@ if (!gadgets.rpc) { // make lib resilient to double-inclusion
       id = String(id);

       // Try window.frames first
-      var target = window.frames[id];
-      if (target) {
-        return target;
-      }
+      // var target = window.frames[id];
+      // if (target) {
+      //   return target;
+      // }

       // Fall back to getElementById()
       target = document.getElementById(id);
@@ -446,7 +446,7 @@ if (!gadgets.rpc) { // make lib resilient to double-inclusion
    * @return {boolean}
    */
     function callSameDomain(target, rpc) {
-      if (typeof sameDomain[target] === 'undefined') {
+      //if (typeof sameDomain[target] === 'undefined') {
         // Seed with a negative, typed value to avoid
         // hitting this code path repeatedly.
         sameDomain[target] = false;
@@ -465,7 +465,7 @@ if (!gadgets.rpc) { // make lib resilient to double-inclusion
           // more meaningful error to the caller.
           gadgets.error('Same domain call failed: parent= incorrectly set.');
         }
-      }
+      //}

       if (typeof sameDomain[target] === 'function') {
         // Call target's receive method
